pkg/controller/supportbundlecollection/controller.go

```diff
@@ -627,6 +696,163 @@ func (c *Controller) isCollectionAvailable(bundleCollection *v1alpha1.SupportBun
 	return true
 }
 
+func (c *Controller) updateStatus(internalBundleCollection *types.SupportBundleCollection) error {
+	updateStatusFunc := func(currentNodes, desiredNodes int, updatedConditions []v1alpha1.SupportBundleCollectionCondition) error {
+		status := &v1alpha1.SupportBundleCollectionStatus{
+			SucceededNodes: int32(currentNodes),
+			DesiredNodes:   int32(desiredNodes),
+			Conditions:     updatedConditions,
+		}
+		klog.V(2).InfoS("Updating SupportBundleCollection status", "supportBundleCollection", internalBundleCollection.Name, "status", status)
+		return c.updateSupportBundleCollectionStatus(internalBundleCollection.Name, status)
+	}
```

Comment:
nit: it seems no benefit gained from having this as a function now.

Reply:
Yes, removed.